libraries/Microsoft.Bot.Schema/ActivityEx.cs

```diff
@@ -141,166 +88,245 @@ public static IConversationUpdateActivity CreateConversationUpdateActivity()
-        public IContactRelationUpdateActivity AsContactRelationUpdateActivity() { return IsActivity(ActivityTypes.ContactRelationUpdate) ? this : null; }
+        /// <returns>IContactRelationUpdateActivity or null.</returns>
+        public IContactRelationUpdateActivity AsContactRelationUpdateActivity()
+        {
+            return IsActivity(ActivityTypes.ContactRelationUpdate) ? this : null;
+        }
 
         /// <summary>
         /// Returns this activity as an installation update activity type; or null, if this is not that type of activity.
         /// </summary>
-        public IInstallationUpdateActivity AsInstallationUpdateActivity() { return IsActivity(ActivityTypes.InstallationUpdate) ? this : null; }
+        /// <returns>IInstallationUpdateActivity or null.</returns>
+        public IInstallationUpdateActivity AsInstallationUpdateActivity()
+        {
+            return IsActivity(ActivityTypes.InstallationUpdate) ? this : null;
+        }
 
         /// <summary>
         /// Returns this activity as a conversation update activity type; or null, if this is not that type of activity.
         /// </summary>
-        public IConversationUpdateActivity AsConversationUpdateActivity() { return IsActivity(ActivityTypes.ConversationUpdate) ? this : null; }
+        /// <returns>IConversationUpdateActivity or null.</returns>
+        public IConversationUpdateActivity AsConversationUpdateActivity()
+        {
+            return IsActivity(ActivityTypes.ConversationUpdate) ? this : null;
+        }
 
         /// <summary>
         /// Returns this activity as a typing activity type; or null, if this is not that type of activity.
         /// </summary>
-        public ITypingActivity AsTypingActivity() { return IsActivity(ActivityTypes.Typing) ? this : null; }
+        /// <returns>ITypingActivity or null.</returns>
+        public ITypingActivity AsTypingActivity()
+        {
+            return IsActivity(ActivityTypes.Typing) ? this : null;
+        }
 
         /// <summary>
         /// Returns this activity as an end of conversation activity type; or null, if this is not that type of activity.
         /// </summary>
-        public IEndOfConversationActivity AsEndOfConversationActivity() { return IsActivity(ActivityTypes.EndOfConversation) ? this : null; }
+        /// <returns>IEndOfConversationActivity or null.</returns>
+        public IEndOfConversationActivity AsEndOfConversationActivity()
+        {
+            return IsActivity(ActivityTypes.EndOfConversation) ? this : null;
+        }
 
         /// <summary>
         /// Returns this activity as an event activity type; or null, if this is not that type of activity.
         /// </summary>
-        public IEventActivity AsEventActivity() { return IsActivity(ActivityTypes.Event) ? this : null; }
+        /// <returns>IEventActivity or null.</returns>
+        public IEventActivity AsEventActivity() => IsActivity(ActivityTypes.Event) ? this : null;
```

Comment:
Why changing only this to lambda expression and not the others?
Can we push this in a different PR as refactor and not as StyleCop changes?
Thanks

Reply:
Thanks for the feedback!